modular_skyrat/modules/gun_quirk/quirk.dm

```diff
@@ -0,0 +1,55 @@
+///Mail goodie list.
+#define GUN_MAIL_LIST list(	/obj/item/gun_maintenance_supplies,\
+							/obj/item/bedsheet/patriot,\
+							/obj/item/clothing/mask/gas/hecu2,\
+							/obj/item/storage/backpack/industrial/cin_surplus/random_color,\
+							/obj/item/storage/belt/military/cin_surplus/random_color,\
+							/obj/item/toy/gun,\
+							/obj/item/toy/ammo/gun,\
+							)
+
+///Gun you get on spawn.
+#define STARTER_GUN_LIST list(/obj/item/gun/ballistic/automatic/pistol/g17,\
+							/obj/item/gun/ballistic/automatic/pistol/mk58,\
+							/obj/item/gun/ballistic/automatic/pistol/pdh/peacekeeper,\
+							/obj/item/gun/ballistic/automatic/dozer,\
+							/obj/item/gun/ballistic/automatic/pistol/cfa_snub,\
+							)
+
+/datum/quirk/item_quirk/gun_nut
+	name = "Gun Nut"
+	desc = "You own a firearm, presumably for self defense or boasting about how cool it is; and you thought it's such a good idea to bring it on station with you."
+	icon = FA_ICON_PERSON_RIFLE
+	value = 6
```

Comment:
I think that 6 is far too low, at minimum I'd make this an 8, and possibly more expensive, I'll have to think on it.
```suggestion
	value = 8
```